plugins/xpay/xpay.c

```diff
@@ -1978,6 +1984,8 @@ static struct command_result *handle_rpc_command(struct command *cmd,
 		json_add_string(response, "maxfee", maxfee);
 	if (partial_msat)
 		json_add_tok(response, "partial_msat", partial_msat, buf);
+	if (maxdelay)
+		json_add_tok(response, "maxdelay", maxdelay, buf);
```

Comment:
Needs a changelog-added comment about how xpay's pay replacement now handles `maxdelay`.

We should probably document fully what cases xpay can handle, probably where we document `xpay-handle-pay` (lightningd-config)

Reply:
> We should probably document fully what cases xpay can handle, probably where we document xpay-handle-pay (lightningd-config)

Good idea! But out of scope for this PR